cli/pcluster/config/param_types.py

```diff
@@ -434,6 +434,26 @@ def get_cfn_value(self):
 # that require some custom action during CFN/file conversion
 
 
+class ExtraJsonParam(JsonParam):
+    """Class to manage extra_json configuration parameters."""
+
+    def to_cfn(self):
+        """Convert param to CFN representation, if "cfn_param_mapping" attribute is present in the Param definition."""
```

Comment:
could you please update the comment to be more specific?
```python
        """
        Convert param to CFN representation.

        The extra_json configuration parameter can contain both "cfncluster" or "cluster" keys but cookbook 
        recipes require "cfncluster" as key.
        """
```

Reply:
updated